mailpoet/tests/integration/_bootstrap.php

```diff
@@ -98,8 +98,9 @@ public function setUp(): void {
     $this->diContainer = ContainerWrapper::getInstance(WP_DEBUG);
     $this->connection = $this->diContainer->get(Connection::class);
     $this->entityManager = $this->diContainer->get(EntityManager::class);
-    // Cleanup scheduled tasks from previous tests and switch cron to Linux method
+    // Cleanup scheduled tasks from previous tests
     $this->truncateEntity(ScheduledTaskEntity::class);
```

Comment:
This was removed and then added back. Is there a reason to keep this? I think it should be safe to remove.

Reply:
Removed: https://github.com/mailpoet/mailpoet/pull/4818/commits/557577a432821dcc8238308bb4ff7b349b3907a4